tests/gnrc_udp/udp.c

```diff
@@ -137,7 +136,11 @@ static void send(char *addr_str, char *port_str, char *data_len_str, unsigned in
         /* add netif header, if interface was given */
         if (iface > 0) {
             gnrc_pktsnip_t *netif = gnrc_netif_hdr_build(NULL, 0, NULL, 0);
-
+            if(netif == NULL) {
+                puts("Error: unable to allocate NETIF header");
+                gnrc_pktbuf_release(payload);
```

Comment:
Shouldn't the `ip` buffer be released instead? (I'm not very familiar with how this works)

Reply:
I tried releasing the IP buffer when I was experimenting and testing with multiple payloads. However, it resulted in Kernel Panic error. I believe resetting the payload cleared the way and allowed the resetting of UDP and IP, and thus the kernel panic error was avoided!
![image](https://user-images.githubusercontent.com/21079172/63352754-4f49a480-c37f-11e9-9861-bcc5d7110b6d.png)